modules/snmp/init.go

```diff
@@ -9,6 +9,19 @@ import (
 
 var snmpHandler = gosnmp.NewHandler
 
+func generateSNMPv3MsgFlags(level int) gosnmp.SnmpV3MsgFlags {
+	var flag gosnmp.SnmpV3MsgFlags
```

Comment:
```suggestion
	flag := gosnmp.NoAuthNoPriv
```

Perhaps make it more explicit?